keel-artifact/src/main/kotlin/com/netflix/spinnaker/keel/artifact/events/KorkArtifactEvent.kt

```diff
@@ -0,0 +1,8 @@
+package com.netflix.spinnaker.keel.artifact.events
+
+import com.netflix.spinnaker.kork.artifacts.model.Artifact
+
+data class KorkArtifactEvent(
```

Comment:
This is a temporary replacement for `ArtifactEvent` which still uses a kork `Artifact` instead of the plugin-friendly `SpinnakerArtifact`. It will be removed in part 3.